roles/bibdata/molecule/default/verify.yml

```diff
@@ -0,0 +1,28 @@
+---
+- name: Verify
+  hosts: all
+  gather_facts: false
+  tasks:
+  - name: check bibdata software status
+    package:
+      name: "{{ item }}"
+      state: present
+    check_mode: true
+    register: pkg_status
+    loop:
+      - cifs-utils
```

Comment:
Is there a reason libmysqlclient-dev & python-mysqldb is not part of the test now?